components/dada-parse/src/parser/code.rs

```diff
@@ -449,7 +449,15 @@ impl CodeParser<'_, '_> {
         } else if let Some((span, token_tree)) = self.delimited('(') {
             let expr =
                 self.with_sub_parser(token_tree, |subparser| subparser.parse_only_expr_seq());
-            Some(self.add(ExprData::Tuple(expr), span))
+
+            Some(self.add(
```

Comment:
For better or worse, I think this change will mean that `(a,)` also parses as a parenthesized expression. We should check that. Not sure if we want that or not!

Reply:
I will investigate the best way to emit an error for that case.

Reply:
Probably we want an error? It's not entirely clear to me.

Reply:
I'd be ok with just opening up an issue to follow-up on later.

Reply:
created #177